spec/features/course/achievement_management_spec.rb

```diff
@@ -59,36 +59,60 @@
                                                           'destroy.success'))
       end
 
-      scenario 'I can award a manually-awarded achievement to a student' do
-        manual_achievement = create(:course_achievement, course: course)
-        auto_achievement = create(:course_achievement, course: course)
-        create(:course_condition_achievement, course: course, conditional: auto_achievement)
+      context 'Award a manually-awarded achievement to student' do
```

Comment:
Let's not use context here, and just adopt 1 large scenarios for now, and test both awarding to my student and to other student, so the test suite does not need to waste time to set up a new test case. You can add some comments within the test to indicate which lines of code are to test awarding to `my_student` and vice versa.